### DISCUSSION TITLE
Never mind. Delete me.

### DISCUSSION THREAD

**Discussion by yaxy** [dlemstra/Magick.NET#854] [Help] [answered]
Never mind. Delete me.

Is there any way to output the lines discovered by HoughLine to a specified MVG file rather than producing an image with the lines?
e.g. HoughLine(iWidth, iHeight, iThreshold, sFileNameMVG)

Or better yet, return an array of strings? e.g.
line 308.142,0 297.896,587  # 204
line 41.674,0 62.1724,587  # 63
line 0,425.627 356,419.413  # 53

Thank you.

**Comment by dlemstra** [comment, marked as answer]
Could you explain why you want me to delete this? Haven't had time to research this earlier. Currently this is not possible but I am thinking of options how we could do that.

**Reply by yaxy** [reply]
Figured it might be difficult. We're going to stick with command line ImageMagick for now.
